test/FastTests/Sparrow/FileSystemUtilsTest.cs

```diff
@@ -0,0 +1,33 @@
+using FastTests.Voron;
+using Sparrow.Platform.Posix;
+using Xunit;
+
+namespace FastTests.Sparrow
+{
+    public class FileSystemUtilsTest : RavenTestBase
+    {
+        [LinuxFact]
+        public void WriteAndReadPageUsingCryptoPager_WhenCall_AllResultShouldBeNotNull()
+        {
+            var swap = KernelVirtualFileSystemUtils.ReadSwapInformationFromSwapsFile();
+            Assert.All(swap, s => Assert.NotNull(s.DeviceName));
+        }
+        
+        [LinuxFact]
+        public void WriteAndReadPageUsingCryptoPager_WhenTwoSwapDefined_AllResultShouldBeNotNull()
+        {
+            var fileName = GetTempFileName();
+            
+            string[] lines =
```

Comment:
Please add also more `lines` to test:
```
{
                "Filename				Type		Size	Used	Priority"
}
```
and
```
{
                "Filename				Type		Size	Used	Priority",
                ""
}

and 
```
{
                "Filename				Type		Size	Used	Priority"
                "/dev/sda                               partition	999420	56320	-2", 
}
```
and also with filename